CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
you can put the COC file in a few places https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-manually - this might help lessen the clutter in the root

Reply:
Same for contribution guidelines

Reply:
i personally dont mind the clutter..